src/main/java/org/mule/extension/aggregator/internal/privileged/executor/AbstractAggregatorExecutor.java

```diff
@@ -201,8 +206,12 @@ public void start() throws MuleException {
     if (clusterService.isPrimaryPollingInstance()) {
       if (!started) {
         startIfNeeded(objectStore);
-        upgradeAggregatedContentIfNeeded();
-        setRegisteredAsyncAggregationsAsNotScheduled();
+        if (failOnStartIfNoQuorum) {
+          upgradeAggregatedContentIfNeeded();
+          setRegisteredAsyncAggregationsAsNotScheduled();
```

Comment:
I would aggregate these 2 calls in another method so that It's clearer that they are the same methods being called int the deferred function.

Reply:
done